services/web-app/next.config.mjs

```diff
@@ -80,43 +80,28 @@ const nextConfig = withMDX(
 
       return config
     },
-    async redirects() {
-      return [
-        // temporarily redirect home page for the first release
-        {
-          source: '/',
-          destination: '/assets',
-          permanent: false
-        },
-        {
-          source: '/assets/:host/:org/:repo/:library',
-          destination: '/assets/:host/:org/:repo/:library/latest',
-          permanent: false
-        }
-      ]
-    },
```

Comment:
@mattrosno  I removed this, it was messing with the asset detail (turning /libraries//carbon-react/latest/assets/accordion into /libraries//carbon-react/latest/accordion/latest)
Do we need this?

Reply:
Everything in your PR looks to be working for me.